### PR TITLE
entity queries now work

### PR DESCRIPTION
**Pull request by samiwh**
#### Description
Docks to go with the entity query pr
#### Documentation
This is it

### CODE REVIEW

File: docs/known-issues.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
...
 | `bool` is not a [blittable type](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types), so will cause an error if used to store component data.                                                                                                                                       | 2018/09/17 | None         | Use `BlittableBool` instead.                                          | No     |
-| Accessing component data from an entity query will segfault, therefore requests where `ResultType` is `SnapshotResultType` will be dropped before sending.                                                                                                                                                                   | 2018/09/19 | None         | Don't send entity queries where `ResultType` is `SnapshotResultType`. | No     |
```
Comment (jessicafalk) [inline docs/known-issues.md]: do we remove the known issues or mark them as resolved? the second one issue is resolved and marked as such, but not removed from the table.

would personally prefer to remove

Comment (firtina-improbable) [reply docs/known-issues.md]: +1 to remove

Comment (samiwh) [reply docs/known-issues.md]: Wait we have a fixed column?

Comment (jessicafalk) [reply docs/known-issues.md]: it is weird. as said offline, let's just remove it

**Pull request merged**